From 02be438d0a1e419ba9b00340c96f2befd815bcb1 Mon Sep 17 00:00:00 2001
From: Tom Finegan <tomfinegan@google.com>
Date: Wed, 23 Sep 2015 11:42:37 -0700
Subject: [PATCH 1/3] build/make/configure.sh: docs for soft_{dis|en}able.

Add function comments explaining what the functions do and do not do.

Change-Id: I23dea09f93bc5cdbea6a0077f90683a1df2f74dc
---
 build/make/configure.sh | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/build/make/configure.sh b/build/make/configure.sh
index fc653f1..46fde74 100644
--- a/build/make/configure.sh
+++ b/build/make/configure.sh
@@ -200,6 +200,10 @@ disabled(){
   eval test "x\$$1" = "xno"
 }
 
+# Iterates through positional parameters, checks to confirm the parameter has
+# not been explicitly (force) disabled, and enables the setting controlled by
+# the parameter when the setting is not disabled.
+# Note: Does NOT alter RTCD generation options ($RTCD_OPTIONS).
 soft_enable() {
   for var in $*; do
     if ! disabled $var; then
@@ -209,6 +213,10 @@ soft_enable() {
   done
 }
 
+# Iterates through positional parameters, checks to confirm the parameter has
+# not been explicitly (force) enabled, and disables the setting controlled by
+# the parameter when the setting is not enabled.
+# Note: Does NOT alter RTCD generation options ($RTCD_OPTIONS).
 soft_disable() {
   for var in $*; do
     if ! enabled $var; then
-- 
2.6.1

